tests/integration/prepare/test_ncep_reanalysis_surface_pr_wtr.py

```diff
@@ -0,0 +1,663 @@
+                        "path": "pr_wtr.eatm.2018.test.nc"
+                    }
+                }
+            },
+            "product_name": "ncep_reanalysis_surface_pr_wtr",
+            "product_type": "auxiliary",
+            "sources": {}
+        },
+        {
+            "creation_dt": "2019-04-17T01:14:42.954377+00:00",
+            "description": "",
+            "extent": {
+                "center_dt": "2018-01-01T18:00:00+00:00",
+                "coord": {
+                    "ll": {
+                        "lat": -91.25,
+                        "lon": -1.25
+                    },
+                    "lr": {
+                        "lat": -91.25,
+                        "lon": 358.75
+                    },
+                    "ul": {
+                        "lat": 91.25,
+                        "lon": -1.25
+                    },
+                    "ur": {
+                        "lat": 91.25,
+                        "lon": 358.75
+                    }
+                }
+            },
+            "format": {
+                "name": "netCDF"
+            },
+            "grid_spatial": {
+                "projection": {
+                    "geo_ref_points": {
+                        "ll": {
+                            "x": -1.25,
+                            "y": -91.25
+                        },
+                        "lr": {
+                            "x": 358.75,
+                            "y": -91.25
+                        },
+                        "ul": {
+                            "x": -1.25,
+                            "y": 91.25
+                        },
+                        "ur": {
+                            "x": 358.75,
+                            "y": 91.25
+                        }
+                    },
+                    "spatial_reference": (
+                        'GEOGCS["Hu Tzu Shan 1950",DATUM["Hu_Tzu_Shan_1950",SPHEROID'
+                        '["International 1924",6378388,297,AUTHORITY["EPSG","7022"]]'
+                        ',TOWGS84[-637,-549,-203,0,0,0,0],AUTHORITY["EPSG","6236"]],'
+                        'PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree"'
+                        ',0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4236"]]'
+                    )
+                }
+            },
+            "id": "1d2b45cd6d00514dbfe54fd35aa16de2",
+            "image": {
+                "bands": {
+                    "water_vapour": {
+                        "band": 4,
+                        "layer": "pr_wtr",
+                        "path": "pr_wtr.eatm.2018.test.nc"
+                    }
+                }
+            },
+            "product_name": "ncep_reanalysis_surface_pr_wtr",
+            "product_type": "auxiliary",
+            "sources": {}
+        },
+        {
+            "creation_dt": "2019-04-17T01:14:42.954377+00:00",
+            "description": "",
```

Comment:
Exclude the description if blank.

Reply:
Here is a CMI link to the precipitable water content.

https://cmi.ga.gov.au/node/373